app/Models/User.php

```diff
@@ -718,6 +718,17 @@ public function setEmailSubscriptionTopicsAttribute($value)
         $this->attributes['email_subscription_topics'] = array_values(array_unique($value));
     }
 
+    /**
+     * Mutator to ensure null is not returned when user re-subscribes.
+     *
+     * @param array $value
+     */
+    public function getEmailSubscriptionTopicsAttribute($value)
+    {
+        //Set empty value equal to array of email_subscription_topics
```

Comment:
Total nit, but maybe something like?
```suggestion
        // Ensure we always return an array value for the email_subscription_topics attribute.
```

(I also wonder if this comment is kinda covered by our method docblock description, but I see this pattern in other method too).

Reply:
sounds good!